py/setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
-exclude = .tox,docs/source/conf.py
+exclude = .tox,docs/source/conf.py,*venv
```

Comment:
pretty common to have a venv (I know we work largely with bazel) but this is something that gets in my way a fair bit, I don't think it hurts to have it excluded in here